manifests/cluster-manager/management/cluster-manager-manifestworkreplicaset-deployment.yaml

```diff
@@ -46,6 +46,7 @@ spec:
         args:
           - "/work"
           - "manager"
+          - "--work-driver=kube"
```

Comment:
add `--cloudevents-client-id=$(POD_NAME)`?

Reply:
currently, we make the `cloudevents-client-id` optional, it only need when we use cloudevents, current case is kube, we can omit it